core/src/main/scala/org/http4s/headers/Content-Disposition.scala

```diff
@@ -79,13 +83,36 @@ object `Content-Disposition` {
         new Renderable {
           def render(writer: Writer): writer.type = {
             writer.append(v.dispositionType)
-            v.parameters.foreach(p => writer << "; " << p._1 << "=\"" << p._2 << '"')
+            v.parameters.foreach(p => writer << "; " << p._1 << "=" << p._2)
```

Comment:
There is a lot of nuance in rendering this correctly.  I think the [akka-http implementation](https://github.com/akka/akka-http/blob/b071bd67547714bd8bed2ccd8170fbbc6c2dbd77/akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/headers.scala#L468-L492) is worth a look.  Note how it renders the extended `filename*` _after_ `filename`, which is important.  It also synthesizes a `filename*` where `filename` is unsafe, which seems like a nice idea.